source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -50,22 +49,12 @@ bool BioXASAppController::startup()
 	dataFolderOK = setupDataFolder();
 
 	// Start up the main program.
-	if (dataFolderOK && AMAppController::startup()) {
-
-		// Initialize singleton objects.
-
-		initializeStorageRing();
-		initializeBeamline();
-		initializePeriodicTable();
-
-		registerClasses();
+	if (dataFolderOK && CLSAppController::startup()) {
 
 		// Ensuring we automatically switch scan editors for new scans.
 		setAutomaticBringScanEditorToFront(true);
 
-		setupExporterOptions();
 		setupScanConfigurations();
```

Comment:
Because `setupScanConfigurations()` is left behind here, and `setupAcquamanUserInterface()` has been promoted to the CLSBeamline, I think views will be created for scan configurations before they exist on BioXAS.